pii_recognition/data_readers/presidio_fake_pii_reader.py

```diff
@@ -0,0 +1,25 @@
+from typing import Dict, List, Set
+
+from pii_recognition.data_readers.data import Data
+from pii_recognition.utils import load_json_file
+
+
+class PresidioFakePiiReader:
+    def _get_supported_entities(self, data: List[Dict]) -> Set:
```

Comment:
are the entities of type `str`?
```suggestion
    def _get_supported_entities(self, data: List[Dict]) -> Set[str]:
```

Reply:
You are right. At this point all three datasets we have are `str` label.